schemas/csdl.schema.json

```diff
@@ -248,7 +254,7 @@
         "@": {
```

Comment:
```suggestion
        "^@": {
```
Sometime you use `"@"`, sometimes `"^@"`. What's the difference?

Reply:
Good catch: the `"@"` is only needed for enumeration members whose value is not an object. Annotations on enum members are prefixed with the member name, for example `"red@Core.Description":"The color red"`

Reply:
The `@` must be preceded by a valid identifier (and not just any valid identifier, but one that identifies a property of the complex type).